modules/apps/data-engine/data-engine-taglib/test/js/data-layout-builder/components/rules/editor/Editor.es.js

```diff
@@ -371,19 +371,30 @@ describe('Editor', () => {
 							fireEvent.click(getByText('value'));
 						});
 
-						await waitFor(() => {
-							document
-								.querySelectorAll('.timeline-item')[1]
-								.querySelectorAll('.ddm-field')[3]
-								.querySelector(selector);
-						});
+						if (type === 'rich_text') {
+							await waitFor(() => {
+								document.querySelectorAll(selector);
+							});
 
-						expect(
-							document
-								.querySelectorAll('.timeline-item')[1]
-								.querySelectorAll('.ddm-field')[3]
-								.querySelector(selector)
-						).toBeTruthy();
+							expect(
+								document.querySelectorAll(selector)
+							).toBeTruthy();
+						}
+						else {
```

Comment:
Hey @rebsilva, I prefer to use early return here to make the code cleaner:

```js
if (type === 'rich_text') {
	expect(
		document.querySelectorAll(selector)
	).toBeTruthy();

	return;
}

await waitFor(() => {
	const richTextElement = document
		.querySelectorAll('.timeline-item')[1]
		.querySelectorAll('.ddm-field')[3]
		.querySelector(selector);
	
	expect(richTextElement).toBeTruthy();
});
```

Reply:
you are right! is indeed better :blush:
i'll test like this to see if this works!

Reply:
Hey @rebsilva, I think you forgot to remove the `else` statement